cns/ipampoolmonitor/metrics.go

```diff
@@ -0,0 +1,70 @@
+package ipampoolmonitor
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	ipamAllocatedIPCount = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "ipam_allocated_ip",
```

Comment:
Might be a good idea to adhere to Prometheus naming conventions [here](https://prometheus.io/docs/practices/naming/). If we do, units should be plural where appropriate.

Reply:
`ip`->`ips` everywhere, since that seems to by my unit here 😆